kube/patch.sh

```diff
@@ -12,7 +12,7 @@ if [  $# -le 1 ]; then
   exit 1
 fi
 
-kubectl -n "${KUBE_NAMESPACE}" patch "$1" "$2" --type strategic --patch "
+kubectl --context="${KUBE_CLUSTER}" -n "${KUBE_NAMESPACE}" patch "$1" "$2" --type strategic --patch "
```

Comment:
This is nice, but what happens if there is no context configured with this name?

Given that this is a sample script, would it be reasonable to document that it needs to be called with the right context (e.g. suggest the user to call kubectl set-context or use gcloud to get cluster credentials)?

Reply:
I've added the context setting command to the README instead :+1: